src/Config.php

```diff
@@ -50,7 +50,7 @@ class Config
         'capture_email',
         'root',
         'scrub_fields',
-        'scrub_whitelist',
+        'scrub_safelist',
```

Comment:
I know the assignment here says `$options` is private, but according to the docs the user has access to set this. Will this change affect existing users?

Reply:
@waltjones, you bring up a good point. I am not totally sure how options are exposed. Let me look into it.

Reply:
Two points:

1. We can _add_ a new key, but we shouldn't _remove_ `scrub_whitelist` at the same time: there should be at least a minor version increment between them.

2. I prefer `allowlist` instead of `safelist`, because the opposite is (IMO) clearer: `denylist` vs `unsafelist`. Is the "safelist" nomenclature a standard in Rollbar?

Reply:
`allowlist` was considered, but the consensus was to standardize on `safelist` (and `blocklist`.)

The CH story for each of the SDKs is to only deprecate, not remove, public facing names until the next major version release.